src/pkg/debug/debug.go

```
@@ -74,19 +74,51 @@ type DebugAgent interface {
+func (d *Debugger) AutoApprove() bool {
+	return d.defaultPermission
+}
+
+// isPaidAccount reports whether the signed-in account is on a paid plan. Any error falls back to
+// false so the prompt stays opt-in.
+func isPaidAccount(ctx context.Context, fabricAddr string) bool {
+	host := client.NormalizeHost(fabricAddr)
+	fabricClient := client.NewGrpcClient(host, client.GetExistingToken(host), "")
+	resp, err := fabricClient.WhoAmI(ctx)
+	if err != nil {
+		term.Debug("Could not determine subscription tier for debug prompt default:", err)
```

golangci-lint (2.12.2)

[error] Error return value of `term.Debug` is not checked

(errcheck)